Ago-Dic-2020/aguilar-cedillo-jonathan-ivan/practica 2/practica-2.py

```diff
@@ -0,0 +1,35 @@
+class Album():
+
+    def __init__(self, titulo, cancion, genero, artista, año):
+        self.titulo = titulo
+        self.cancion= cancion
+        self.genero = genero
+        self.artista = artista
+        self.año = año 
+
+    def get_titulo(self):
+        return "Informacion: \nTitulo: {} \ncancion: {} \ngenero: {} \nartista: {}\naño: {}\n".format(self.titulo, self.cancion, self.genero, self.artista, self.año)
+
+    def get_cancion(self):
+        return"resumen: \ncancion: {}, \ngenero: {}, \nartista: {} \n".format(self.cancion, self.genero, self.artista)
+    
+class Cancion(Album):
+    
+    def __init__(self, titulo, cancion, genero, artista, año, Nombre, pista, duracion):
+        Album.__init__(self, titulo, cancion, genero, artista, año)
+        self.Nombre= Nombre
```

Comment:
Las variables y funciones deben ser estilo `snake_case` de acuerdo al estándar oficial: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names